Documents/conversions/convert_sqlite.md

```diff
@@ -0,0 +1,145 @@
+# Converting an SQLite Table to other forms
+
+## From SQLite to PostgreSQL
+### Inputs
+* path to sqlite file
+* table name
+* PostgreSQL schema name
+* PostgreSQL table name (defaults to sqlite tablename)
+
+### PostgreSQL prerequisites
+
+The database is called `desc_dc2_drp`.  In order to make changes to the database you must have access to the account `desc_dc2_drp_admin`.  By default PostgreSQL applications like the command-line tool psql will look for account information  in the file `.pgpass` in your home directory, readable (and writable) only by you.  Then you can invoke psql as follows:
```

Comment:
I wonder if you want to leave a pointer as to how to access PostgreSQL in DESC, maybe link to https://confluence.slac.stanford.edu/display/LSSTDESC/Getting+Started+with+PostgreSQL+at+NERSC ?